flask_monitoringdashboard/static/pages/configuration.html

```diff
@@ -43,6 +35,97 @@
     </div>
 </div>
 
+<modal name="'delete'" title="'Delete user'">
```

Comment:
For these kinds of confirmations, I would personally just use something like [sweetalert](https://sweetalert2.github.io/) or if I am feeling really lazy (90% of my projects), just roll with:

```js
window.confirm("Are you sure you want to delete this user?") && deleteUser();
```

Reply:
I would like to keep the modal, because it looks better 😅